Packs/CommonScripts/Scripts/PcapHTTPExtractor/PcapHTTPExtractor.py

```diff
@@ -1,10 +1,30 @@
-import demistomock as demisto
 from CommonServerPython import *
 from CommonServerUserPython import *
 import zlib
 import pyshark
 from datetime import datetime
 import re
+import sys
+import traceback
+try:
+    from StringIO import StringIO  # for Python 2
+except ImportError:
+    from io import StringIO  # for Python 3
+
+serr = sys.stderr
+sys.stderr = StringIO()
+
+''' GLOBAL VARIABLES '''
+LIMIT = demisto.args()["limit"]
+START = demisto.args()["start"]
+LIMIT_DATA = int(demisto.args()["limitData"])
+if "allowedContentTypes" not in demisto.args():
+    ALLOWED_CONTENT_TYPES = ("text", "application/json", "multipart/form-data",
+                             "application/xml", "application/xhtml+xml",
+                             "application/ld+json", "application/javascript",
+                             "multipart/alternative", "application/x-www-form-urlencoded")
+else:
+    ALLOWED_CONTENT_TYPES = tuple(demisto.args()["allowedContentTypes"].split(","))  # type: ignore
```

Comment:
move this block to the main function:
* define the globals here with default values
* populate them in the main function with the desired values